kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README

```diff
@@ -0,0 +1,55 @@
+This sample has been enhanced to provide the ability for a user to introspect an already existing domain
+(e.g. on-prem domain) using the WebLogic Deploy Tooling (WDT) and then create the same domain on Persistent Volume (PV)
+in Kubernetes using this sample. The WDT Discover tool will generate a WDT model yaml (metadata representation of the
+domain configuration) and a properties file which will contain a key=value for each property in the WDT model.
+The sample has been enhanced to update the domain using both WDT Update and the WDT Extract Domain Resource Tools.
+
+New functionality for the Domain Home on PV sample:
+1- The create domain script create-domain.sh will use the WDT Extract Domain Resource Tool to generate the domain CR yaml (domain.yaml).
+2- The ability to update the domain on PV which was originally created as described above using WDT. The domain
+   configuration will be updated by running WDT Update Tool using a WDT full model and the equivalent change to
+   the Domain CR using the WDT Extract Domain Resource Tool.
+3- The create and update domain will accept the following inputs: a WDT model, and the WDT properties file
```

Comment:
The create and update domain ->  (do you mean) The Create Domain Tool and Update Domain Tool (If so, use the complete tool name)
will accept -> accepts
inputs: -> input:
model, and -> model and (no comma) (Do you mean respectively? In other words the Create Domain Tool accepts a WDT model and the Update Domain Tool accepts the WDT properties file. If so, the add , (comma) respectively.